app/templates/Articles/add.php

```diff
@@ -0,0 +1,10 @@
+<h1>Add Article</h1>
+<?php
+    echo $this->Form->create($article);
+    // @TODO: Update this when the authorizacion was implemented
+    echo $this->Form->control('user_id', ['type' => 'hidden', 'value' => 1]);
+    echo $this->Form->control('title');
+    echo $this->Form->control('body', ['rows' => '3']);
+    echo $this->Form->button(__('Save Article'));
```

Comment:
In a project where we've to support multilanguage, the usage of `__()` function is really important. Please refer https://book.cakephp.org/4/en/core-libraries/internationalization-and-localization.html

Let's say we've a project that didn't plan to support the multilanguage yet; Do you think it's necessary to use `__()` function?

Reply:
It is not necessary but it is a good practice to use it from the beginning, because if in the future you want to add internalization it will be easier